frontend/src/styles/_uswds-theme-custom-styles.scss

```diff
@@ -21,3 +21,30 @@ i.e.
 */
 
 @use "uswds-core" as *;
+
+h2 {
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 40.22px;
+}
+
+h3 {
+  font-size: 1.375rem;
+  line-height: 26.75px;
+}
+
+p {
+  font-weight: 400;
+  line-height: 25.92px;
+}
+
+.grid-container {
+  padding: 1rem 0;
+}
+
+.usa-intro {
+  font-weight: 300;
+  font-size: 22px;
+  line-height: 35.64px;
```

Comment:
I get being pixel-perfect, but these decimal values might make mathing and inspecting and aligning things more difficult or tedious later. We can change it then, but maybe round up or down? I'm not even sure to what extent a browser respects complicated decimal pixels. Googling...

Reply:
Yeah, I agree that the values are weird. I'm using the ones set by Andy in the Figma design doc. 
I'm thinking we can wait till monday when he gets back to ask him what he thinks.